scripts/migration/migrate_personal_to_profile_websites.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script to migrate nodes with invalid categories."""
+
+import logging
+
+from modularodm import Q
+from nose.tools import *
+from tests.base import OsfTestCase
+
+from website import models
+from website.app import init_app
+from scripts import utils as scripts_utils
+from tests.factories import AuthUserFactory
+
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    init_app(routes=False)
+    logger.info("migrating personal to profileWebsites")
+            
+    for user in get_users_with_social_field():
+        logger.info(repr(user))
```

Comment:
This is very verbose logging (you're logging every user who has set a social field. Can you make this more useful? Also, better to make the logging more human-readable, e.g. "Migrating user: {user_id}" rather than just the repr of the user.